build/scripts/coverage.sh

```diff
@@ -5,8 +5,8 @@
 #
 # Code coverage generation
 CVG_EXCLUDE="${CVG_EXCLUDE:-test}"
-
-go test -coverpkg=./... -coverprofile ./coverage.cov $(go list ./... |  grep -Ev "${CVG_EXCLUDE}")
+go test -coverpkg=./... -coverprofile ./coverageTmp.cov $(go list ./... |  grep -Ev "${CVG_EXCLUDE}")
```

Comment:
If we keep this approach, a comment of what is being accomplished here would be helpful.